### DISCUSSION TITLE
Errors in Pi-hole admin interface

### DISCUSSION THREAD

**Discussion by neitsab** [chadgeary/cloudblock#103] [Q&A]
During my first connection to Pi-hole admin and on all subsequent connections, I see the following error when accessing the Settings menu:

```php
PHP error (2): parse_ini_file(/etc/pihole/setupVars.conf): failed to open stream: Permission denied in /var/www/html/admin/
settings.php:17
```
![screenshot](https://github.com/chadgeary/cloudblock/assets/2963014/6267deb5-96eb-4957-83dc-febe16231214)

Researching it led me to [this thread](https://discourse.pi-hole.net/t/php-permissions-error-web-admin-page/14250/4) asking the result of `ls -al /etc/pihole/setupVars.conf`:

```sh
ubuntu@cloudblock-instance:~$ sudo docker exec -it pihole  ls -al /etc/pihole/setupVars.conf
-rw-r--r-- 1 1000 root 215 Dec 14 22:15 /etc/pihole/setupVars.conf
```

UID 1000 is `opc` on Oracle Cloud:

```
ubuntu@cloudblock-instance:~$ id 1000
uid=1000(opc) gid=1000(opc) groups=1000(opc)
```

However Pi-hole the "System" page of the Settings do give `root` as User/Group so... Dunno ¯\_(ツ)_/¯

Another error appears anytime I open Groups:

```
DataTables warning: table id=groupsTable - Invalid JSON response. For more information about this error, please see http://datatables.net/tn/1
```
![image](https://github.com/chadgeary/cloudblock/assets/2963014/ae56f88e-b1af-4b3f-8b8d-8ac8b8e7c276)

Has anyone seen those?

**Comment by chadgeary** [comment]
if you have a moment, give it a try after this change: https://github.com/chadgeary/cloudblock/pull/104
